neo/VM/ExecutionEngine.py

```diff
@@ -246,8 +238,7 @@ def ExecuteOp(self, opcode, context):
                 n = estack.Pop().GetBigInteger()
 
                 if n < 0:
-                    self._VMState |= VMState.FAULT
-                    return
+                    return self.VM_FAULT_and_report(VMFault.UNKNOWN3)
```

Comment:
Will `UNKNOWN3-6` be replaced with something?

Reply:
For now not, unless you have a clear idea what a good fault statement would be for those. I decided to keep them `generic unknown` until with some more experience I figure out a good name for it.